web_src/less/markup/asciicast.less

```diff
@@ -0,0 +1,10 @@
+@import "../asciinema-player/dist/bundle/asciinema-player.css";
```

Comment:
The JS is loaded async now, the CSS should also be loaded async ....

Reply:
Could you please show me how to do that?

Reply:
Just add it to the JS as a second import. I will put a suggestion.